### PR TITLE
MF-389 - Add documentation for Writers and Readers

### PR DESCRIPTION
**Pull request by blokovi**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -327,3 +327,125 @@ mosquitto_sub -u <thing_id> -P <thing_key> -t channels/<channel_id>/messages -h
 
 If you are using TLS to secure MQTT connection, add `--cafile docker/ssl/certs/ca.crt`
 to every command.
+
+## ADDONS
```
Comment (drasko) [inline docs/getting-started.md]: Why is this allcap? Not consistent with other headlines.

File: writers/influxdb/README.md [hunk 2]
```diff
@@ -63,12 +63,12 @@ MF_NATS_URL=[NATS instance URL] MF_INFLUX_WRITER_PORT=[Service HTTP port] MF_INF
 ### Using docker-compose
 
 This service can be deployed using docker containers.
-Docker compose file is available in <project_root>/docker/addons/influxdb/docker-compose.yml. Besides database
+Docker compose file is available in <project_root>/docker/addons/influxdb-writer/docker-compose.yml. Besides database
```
Comment (drasko) [inline writers/influxdb/README.md]: Use backticks to verbatim-format the code.

**Review by drasko — CHANGES_REQUESTED** [review]
Very good doc here @blokovi. Just a few minor remarks from my side.

**Review by drasko — DISMISSED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**